src/components/QrcodeRenderer.vue

```diff
@@ -68,4 +76,7 @@ img {
   display: block; // [1]
   max-width: 100%;
 }
+.spinner-container {
+  overflow: hidden;
```

Comment:
what is the purpose of overflow: hidden?

Reply:
Without it there's a horizontal scroll while spinner is spinning, and it kinda jumps because spinner is, well, spinning and it has different width every moment and therefore one moment there is a scroll and the other there is not

Reply:
I cant reproduce this issue. If you still do, try to find another solution. I don't think the `overflow: hidden` is the right way to solve it.

Reply:
With your styles I can't reproduce this issue either. Or maybe it was because of the opened dev tools